index.html

```diff
@@ -15,14 +15,254 @@
+          </div>
+          <div class="col-md-4 col-12 feature-card-3 wow fadeInUp" data-wow-delay="0.7s">
+            <!-- start card 3 -->  
+            <div class="feature-card">
+              <div class="feature-card-icon">
+                <img src="img/service-icon-3.png"/>
+              </div>
+              <h3>Totally Free</h3>
+              <p>Writers and stars of Veep have responded incredulously to the news an Australian politician required stitches way.</p>
+            </div>
+            <!-- end card 3-->
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- end feat: feature cards -->
+    <!-- END ABOUT FEAT#13, BUILT BY MARCIN WICHA-->
+    
+    <!-- START ABOUT FEAT#14, BUILT BY MARCIN WICHA-->
+    <!-- START SOCIAL AND HEIGH QUALITY CONTAINER -->
+    <div class="container">
+      <div class="row section-social-features">
+        <!-- start social features photo -->
+        <div class="col-sm-6 d-none d-md-block social-photo">
+          <img class="img-fluid" src="img/mobile-1.png" alt="Photograph of a mobile phones social network features."/>
+        </div>
+        <!-- end social features photo -->
+        <!-- start social features column -->
+        <div class="col-12 col-sm-6 social-feature-col">
+            <!-- start lunatic icon -->
+            <div class="social-feat-icon wow fadeInUp" data-wow-delay="0.3s">
+              <img src="img/about-logo.png" alt="Lunatic's logo"/>
+            </div>
+            <!-- end lunatic icon -->
+            <!-- start features-text -->
+            <h3 class="social-feat-heading wow fadeInUp" data-wow-delay="0.5s">Integrated Social Network</h3>
+            <p class="social-feat-text wow fadeInUp" data-wow-delay="0.7s">You can now use all Social Network from this Lunatic app also. Writers and stars of Veep have responded incredulous to the news an Australian politician required stitches after knocking himself unconscious while laughing.</p>
+            <!-- end features-text -->
+            <!-- start stats-box-1 -->
+            <div class="social-stats-box stats-box-1 d-inline-block wow fadeInUp" data-wow-delay="0.9s">
+              <h4 class="stats-box-heading">2.5 B</h4>
+              <p class="stats-box-text">User</p>
+            </div>
+            <!-- end stats-box-1 -->
+            <!-- start stats-box-2 -->
+            <div class="social-stats-box stats-box-2 d-inline-block wow fadeInUp" data-wow-delay="1.1s">
+                <h4 class="stats-box-heading">2.6 B</h4>
+                <p class="stats-box-text">Download</p>
+            </div>
+            <!-- end stats-box-2 -->
+          </div>
+        <!-- end social features column --> 
+      </div>
+      <!-- END ABOUT FEAT#14, BUILT BY MARCIN WICHA-->
+      <!-- START ABOUT FEAT#15, BUILT BY MARCIN WICHA-->
+      <div class="row section-social-features">
+        <!-- start video features column -->
+        <div class="col-12 col-sm-6 social-feature-col">
```

Comment:
Bug: Content should resize on medium screen,
```suggestion
        <div class="col-12 col-md-6 social-feature-col">
```